game/templates/game/replay.html

```
@@ -0,0 +1,100 @@
+                <div id="moveHistoryBody" style="overflow-y: auto; flex-grow: 1;">
+                    <p style="color: #666; text-align: center; margin-top: 20px;">Load a match to see history.</p>
+                </div>
+            </div>
+            <div class="card data-load-card">
+                <button id="btnFetchDB" class="btn btn-gold"
+                    style="width: 100%; background: #4a90e2; color: #fff; border-color: #4a90e2;">Watch Recent
+                    Match</button>
+                <div style="margin-top: 15px;">
+                    <label style="font-size: 0.8em; color: #aaa; display: block; margin-bottom: 5px;">Or upload a PGN
+                        file:</label>
+                    <input type="file" id="pgnUpload" accept=".pgn"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)